### PR TITLE
barre de recherche admin entreprise

### PR DESCRIPTION
**Pull request by endricklosange**

### CODE REVIEW

**Review by maxime-guichard — APPROVED** [review]

File: src/Controller/AdminController.php [hunk 1]
```diff
@@ -22,15 +26,16 @@ public function index(): Response
...
-    public function companiesList(): Response
+    public function companiesList(CompanyRepository $repository, Request $request): Response
     {
-        $companies = $this->getDoctrine()
-            ->getRepository(Company::class)
-            ->findAll();
+        $data = new SearchData();
+        $form = $this->createForm(SearchType::class, $data);
+        $form->handleRequest($request);
+        $companies = $repository->findsearch($data);
```
Comment (LucyMerlier) [inline src/Controller/AdminController.php:34]: "findSearch" => in camelCase

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by maxime-guichard — APPROVED** [review]

File: src/Data/SearchData.php [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace App\Data;
+
+class SearchData
+{
+    public ?string $searchQuery = '';
+}
```
Comment (LucyMerlier) [inline src/Data/SearchData.php:8]: no getters and setters, just a public property?

File: src/Form/SearchType.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+class SearchType extends AbstractType
+{
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('searchQuery', TextType::class, [
+                'label' => false,
```
Comment (LucyMerlier) [inline src/Form/SearchType.php:18]: if you don't use a label, you should add an "aria-label" attribute to your input for accessibility

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by JuMn88 — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**